doc/man3/SSL_get_rpoll_descriptor.pod

```diff
@@ -2,8 +2,8 @@
 
 =head1 NAME
 
-SSL_get_rpoll_descriptor, SSL_get_wpoll_descriptor, SSL_net_read_desired,
-SSL_net_write_desired - obtain information which can be used to determine when
+SSL_get_rpoll_descriptor, SSL_get_wpoll_descriptor
```

Comment:
These functions also just make sense for quic, so this whole file can be deleted along with them.

Reply:
what about the `BIO_get[rw]poll_descriptor` functions?